process/subgroups.md

```diff
@@ -0,0 +1,68 @@
+| SIMD  | Petr Penzin (@penzn), Ng Zhi An (@ngzhian)  | ? | [Agendas](https://github.com/WebAssembly/meetings/tree/main/simd/)  |
+| Stack Switching  | Francis McCabe (@fgmccabe)  | [Repo](https://github.com/WebAssembly/stack-switching)  | [Agendas](https://github.com/WebAssembly/meetings/tree/main/stack)  |
+| WASI  | Lin Clark (@linclark), Sam Clegg (@sbc)  | [Repo](https://github.com/WebAssembly/WASI)  | [Agendas](https://github.com/WebAssembly/meetings/tree/main/wasi)  |
+
+## Subgroup Leadership Expectations
+
+Subgroups require two kinds of leadership. These roles can be filled by the same person or multiple people:
+
+- Chair(s)—responsible for running the group
+- Champion(s)—responsible for driving technical progress on proposals
+ 
+### Chair
+A subgroup chair is expected to:
+
+- Organize subgroup meetings:
+  - Create agendas in advance so that others can add issues via PRs. These agendas should live in a subgroup-specific folder in the [WebAssembly meetings repo](https://github.com/WebAssembly/meetings). Alternatively, agendas can be organized via issues on a proposal repo for subgroups covering just a single proposal.
+  
+      Each agenda should have a header that includes meeting details and instructions for how to join. See [this agenda](https://github.com/WebAssembly/meetings/blob/main/wasi/2021/WASI-10-07.md) for an example.
+  
+      Note: If you use a Google form for registrations, ensure that you turn on email notifications:
+      - Go to the "Responses" tab in the editing view
+      - Click the "..." menu button in the top pane
+      - Turn on "Get email notifications for new responses"
+
+  - Add [new participants](https://www.w3.org/community/webassembly/participants) to the Google calendar event. You should ensure the participant is a CG member before adding them by searching for their name in the participant list. If they are not yet listed, explain that registration is free and that they will need to complete that first.
+
+  - While not required, it can be helpful to work 1:1 with champions to ensure they are identifying topics for discussion at meetings and adding them to the agenda.
+
+- Run subgroup meetings:
+
+  - Ensure notes are taken and posted to the meetings repository within one week post-meeting. Public notes are a requirement of the W3C.
+
+  - Facilitate the discussion and step in when disagreements are getting heated or one person is dominating the conversation in an unproductive way.
+
+  - Identify and drive consensus among participants.
+
+  - Identify action items and make note of commitments.
+
+  - Enforce the CoC (see below).
+
+- Maintain the issue queue:
+  - Triage new issues as they come in. This means different things for different proposals, but as the chair, you should keep an eye on issues as they come in and try to ensure that relevant issues are responded to appropriately.
```

Comment:
if the issue is for a proposal (rather than for the subgroup itself) maybe this should be the job of the champion?  In the case of e.g. WASI there's one repo for the group and another for proposals, but for GC, it's basically the same repo for both. I guess that's what you mean by saying it means different things for different proposals. Maybe it could be clearer as "this means different things for different subgroups (e.g. whether the group has a repo that's independent of any single proposal)" or something like that.

Reply:
I clarified the wording here with 0ff2c86